lib/components/breadcrumbs/breadcrumbs.less

```diff
@@ -25,12 +25,16 @@
         margin-top: var(--su2);
     }
 
-    & &--link {
-        color: var(--_br-link-fc);
-
+    & &--link,
+    & a&--link{
```

Comment:
I am not sure I understand what the `a` does here? `as-breadcrumbs--link`?

Reply:
This `&` in less always just plugs in the selector found in the outer scope, so in this case, it's targeting:

```css
  .s-breadcrumbs a.s-breadcrumbs--link
```

The `a` is just there to ensure it applies to breadcrumbs that are links.